Sources/Property.swift

```diff
@@ -366,6 +366,23 @@ extension PropertyProtocol {
 	}
 }
 
+extension PropertyProtocol where Value == Bool {
+	/// Returns a property with the oposite value
+	var not: Property<Value> { return self.map { !$0 } }
+	
+	///Combines the property with the given property in a manner described by
+	///`and()` operator.
+	func and(_ property: Property<Value>) -> Property<Value> {
+		return self.combineLatest(with: property).map { $0 && $1 }
```

Comment:
You mean to do something like this?
```
public func and(_ newProducer: SignalProducer<Bool, NoError>) -> Property<Value> {
		return self.lift<Bool> { (producer: SignalProducer<Value, NoError>) in
			producer.combineLatest(newProducer).map(&&)
		}
	}
```

using `lift<U>` grom the `PropertyProtocol`?
```
fileprivate func lift<U>(_ transform: @escaping (SignalProducer<Value, NoError>) -> SignalProducer<U, NoError>) -> Property<U> {
		return Property(self, transform: transform)
	}
```

If thats the case i don't see the point to use it when we can use `combineLatest`